scripts/purge.py

```diff
@@ -504,46 +537,55 @@ def get_all_tagged_arns(filter_tags):
 if args.do:
     log.info(f"Only processing types: {args.do}")
 
-for resource in resources:
+try:
+  for resource in resources:
```

Comment:
Is it intentional that any failure triggers an exit of the script, rather than moving on?  

IIRC, we'd frequently see failures from like `IAM` or `VPC` to fail to  delete, but we'd push on.